### ISSUE TITLE
Google Translate strings still used in Lao version of app

### ISSUE THREAD

**Issue by Abby-Wheelis** [e-mission/e-mission-docs#943]
As we were first getting a set of translation files prepared in Lao, the set of translations provided had a few gaps, which we filled using Google Translate which means those places require careful review. This issue tracks the progress of finalizing translations for the Lao version of the app. 

A round of feedback from the team working on the Lao translations has just been implemented. We're waiting on review and correction of a few key places:

- [x] modes and purposes in [`trip_confirm_options-lo.json`](https://github.com/e-mission/e-mission-translate/blob/master_for_platform/lo/i18n/trip_confirm_options-lo.json)
- [x] join page related translations in [`lo.json`->'join'](https://github.com/e-mission/e-mission-translate/blob/d0f1c14d10381b54bbd43f16c32db331d6583a52/lo/i18n/lo.json#L444)
- [x] the [survey](https://github.com/e-mission/e-mission-phone/blob/framework_migration_prep/survey-resources/data-xls/demo-survey-v2.xlsx) has a blank section pending a decision about what would be contextually appropriate units of currency 

See prior activity about setting up the app in Lao in #938

**Comment by Abby-Wheelis** [comment]
We now have a meeting scheduled for 9am MT on Wed Aug 9th to clarify the remaining translations. I plan to discuss the three points listed above as well as any other concerns that team has.

**Comment by Abby-Wheelis** [comment]
In the meeting this morning, @shankari and I walked through the screens of the app and the join page with the team working on translations. We identified places in the Join webpage, modes and purposes, permissions screen, and dashboard where the Lao translations just did not make sense. I have sent highlighted files back to person working on translations to point out where in the code these phrases are controlled. 

We also identified the need for an adapted survey due to differences in cultural context. In particular, income brackets and ethnicity identifiers are different in Laos than in the US, so we're also in the process of polishing a custom survey. The survey is also set up to handle the difference between 2 and 3/4 wheeled vehicles since 2 wheeled vehicles are a more common transportation mode, and statistics about access to such vehicles could provide good insight.

**Comment by Abby-Wheelis** [comment]
I have more feedback about translations, and am working to incorporate those changes this morning. I'm running into a few changes that they've requested that we can't make:

- changing the goals displayed on the dashboard: they're pointing out that having the US goals doesn't make much sense for their context -> not sure that we can change this now, but possibly something to keep in mind for eventual modularity of the dashboard. 
- translating the contact info -> right now this is read from the dynamic config, but not from the translated section of the dynamic config.

**Comment by Abby-Wheelis** [comment]
One adjacent request that they made which we could fill is to add a line break between the program title and the words "Join Page" which would look like this:
![Screenshot 2023-08-10 at 10 43 41 AM](https://github.com/e-mission/e-mission-docs/assets/54848919/4356a6b8-db11-48a0-92b1-513b7b1f9e85)

![Screenshot 2023-08-10 at 10 43 35 AM](https://github.com/e-mission/e-mission-docs/assets/54848919/689a2757-2ef0-4a3c-a92e-2b87aae0d3e1)

Any opinions on how this looks?

**Comment by shankari** [comment]
@Abby-Wheelis I am not the best person to ask about what looks good, what do you think?

**Comment by Abby-Wheelis** [comment]
@shankari The contact info I'm talking about is what's listed in the dynamic config file - for `program_admin_contact` and that key is not currently set up to have versions in multiple languages. That key, from a quick scan, is used for the join webpage as well as in the app on the consent screen. This is something we might want to revise at some point, as it stands the person's name and contact are all one string, but it might be best to split it up -- that would also lend to having words like "email" and "phone" translated, but only inputting the contact info in one place.

**Comment by Abby-Wheelis** [comment]
@shankari The line break looks a little awkward to me, but a visual separator between "name" and "join page" seems like a good idea. I tried "-" and ":" and I like the way ":" looks and it still looks good even if there happens to be a line break
![Screenshot 2023-08-10 at 11 12 01 AM](https://github.com/e-mission/e-mission-docs/assets/54848919/64a5dcc6-18f2-49a0-aeca-fb50005c5247)
![Screenshot 2023-08-10 at 11 12 08 AM](https://github.com/e-mission/e-mission-docs/assets/54848919/84976be4-6f95-4bed-b749-46e058e701da)

**Comment by shankari** [comment]
ah I remember setting it up that way, and making the decision that we didn't need to internationalize the name (at least between English and Spanish), since the name, email and phone are not really language-specific.

But I guess this can be important for other scripts - e.g. K. Shankari versus க. சங்கரி versus क. शंकरी 

Is their ask to have the name in the Lao script? If so, I would suggest that we have the `program_admin_contact` only in Lao for now and expand this in the next release.

**Comment by Abby-Wheelis** [comment]
They did not ask specifically to have the name put into Lao, just provided a translation for the section which in English is along the lines of "For questions about the program, please contact First Last, National Renewable Energy Laboratory (email.email@email.com), phone: (###-###-####)" which, now that I actually compare that to other configs, is excessive. It looks like others at most say "name, organization at (email.email@email.com) or (###-###-####)"

Should I trim down the English and get back to them? I know that puts us on more of a delay, but there is also one more string that I found that is still in English, so I probably need to reach out about that anyways.

**Comment by shankari** [comment]
I think that the "For questions about the program" part is in the template -e.g.
The program_admin_contact should only have name, email, phone

```
<h3>Questions</h3>
If you have any questions about the data collection goals and results, please
contact the primary point of contact for the study,
{{ui_config.intro.program_admin_contact}}. If you have any technical questions
about app operation, please contact NREL’s K. Shankari (k.shankari@nrel.gov).
```

**Comment by Abby-Wheelis** [comment]
Related to the survey and the race/ethnicity question, I got back the three options we spoke about in the call, but also translations for the other options we had listed. @shankari do you recall if they wanted to replace all of the choices with just their three, or make a swap?

The way I'm interpreting it, the final list could include east asian, south asian, american indan or alaskan native, and native hawaiian or pacific islander or exclude those options. White, Black, and Latinx seem to be overwritten by the contextually specific options and I think it makes sense to keep other, multiracial, and prefer not to say around as options.

**Comment by shankari** [comment]
@Abby-Wheelis replace. So they will have the three ethnic groups, other, multiracial and prefer not to say. We should drop the extras.

**Comment by shankari** [comment]
> there is also one more string that I found that is still in English, so I probably need to reach out about that anyways.

which string was it? I wonder if we could ping Kosol to fix it - we can send him the Google Translate to help

**Comment by Abby-Wheelis** [comment]
"We will use the " in join page's lo.json. I'm working on an email right now to ask about that and the preferred language for a contact, and a place where the `{{code_var}}` type keys are getting things a little lost in translation.

Are we removing the old keys now or next release? There's a translation that's a blend of Lao and English right now, and another with a substitution that seems off but both on their way out - I suppose we don't have to correct it either way since it's unused.

**Comment by Abby-Wheelis** [comment]
At this point as far as I know, we're just waiting on a translation for the one string in the join page, and a decision on what language to use for the contact information in the config.

**Comment by shankari** [comment]
Resolved in https://github.com/e-mission/nrel-openpath-join-page/pull/22 and https://github.com/e-mission/e-mission-translate/pull/39
